suites/pacific/rbd/tier-2_rbd_regression.yaml

```diff
@@ -161,3 +161,18 @@ tests:
       module: rbd_snapshot_rename_advanced.py
       name: Test Snapshot Rename with clone operations in progress
       polarion-id: CEPH-9836
+
+  - test:
+      desc: Trying to delete a protected snapshot should fail
+      config:
+        do_not_create_image: True
+        operations:
+          map: true
+          io: true
+          nounmap: true
+      module: rbd_delete_protected_snapshot_krbd.py
+      name: krbd client - Test to Perform Deletion of protected snapshot
+      polarion-id: CEPH-9224
+
```

Comment:
one small change , extra lines at the end of the file.
Other changes look good

Reply:
done